content/applications/inventory_and_mrp.rst

```diff
@@ -13,3 +13,4 @@ Inventory & MRP
     inventory_and_mrp/barcode
     inventory_and_mrp/quality
     inventory_and_mrp/maintenance
+    inventory_and_mrp/plm
```

Comment:
Small thing I noticed:
The toctree links in this doc aren't exactly lined up with the indentation of the tag 😄